### PR TITLE
Device manager - device security recommendation card (PSG-637)

### PR DESCRIPTION
**Pull request by kerryarchibald**
Screenshot 2022-08-09 at 15 33 34
Screenshot 2022-08-10 at 17 32 18
Screenshot 2022-08-10 at 17 25 24

- Deprecates e2e `warning` and `unverified` svgs that use hardcoded colors, to avoid large refactor
- adds new `warning` and `unverified` svgs that use `currentColor`
- add light variant of warning and verified colours
- adds new device security recommendation card

~NB: these are not used yet, to avoid blocking this PR on #9151~

## Checklist

* [ ] Tests written for new code (and old code if feasible)
* [ ] Linter and other CI checks pass
* [ ] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md))

Notes: Device manager - device security recommendation card

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: res/themes/light/css/_light.pcss [hunk 1]
```diff
@@ -201,6 +201,8 @@ $e2e-verified-color: #76cfa5; /* N.B. *NOT* the same as $accent */
 $e2e-unknown-color: #e8bf37;
 $e2e-unverified-color: #e8bf37;
 $e2e-warning-color: #ba6363;
+$e2e-verified-color-light: #f1faf9;
+$e2e-warning-color-light: #fff5f5;
```
Comment (SimonBrandner) [inline res/themes/light/css/_light.pcss:205]: Is this based on one of the compound colors?

File: src/components/views/elements/AppWarning.tsx [hunk 2]
```diff
@@ -24,7 +24,7 @@ const AppWarning: React.FC<IProps> = (props) => {
     return (
         <div className='mx_AppPermissionWarning'>
             <div className='mx_AppPermissionWarningImage'>
-                <img src={require("../../../../res/img/warning.svg").default} alt='' />
+                <img src={require("../../../../res/img/e2e/verified.svg").default} alt='' />
```
Comment (SimonBrandner) [inline src/components/views/elements/AppWarning.tsx:27]: This should be warning, right?

File: src/components/views/right_panel/EncryptionPanel.tsx [hunk 3]
```diff
@@ -85,7 +85,7 @@ const EncryptionPanel: React.FC<IProps> = (props: IProps) => {
         // handle transitions -> cancelled for mismatches which fire a modal instead of showing a card
         if (request && request.cancelled && MISMATCHES.includes(request.cancellationCode)) {
             Modal.createDialog(ErrorDialog, {
-                headerImage: require("../../../../res/img/e2e/warning.svg").default,
+                headerImage: require("../../../../res/img/e2e/verified-deprecated.svg").default,
```
Comment (SimonBrandner) [inline src/components/views/right_panel/EncryptionPanel.tsx:88]: This should be warning, right?

**Review by SimonBrandner — COMMENTED** [review]
Looking sane though I am confused about the icon changes

File: res/themes/light/css/_light.pcss (hunk 1 above)
Comment (kerryarchibald) [reply res/themes/light/css/_light.pcss:205]: These are based on new designs from Jano on figma, which I assume are based on compound

Comment (SimonBrandner) [reply res/themes/light/css/_light.pcss:205]: Then we should definitely be calling these what they are called in Compoud. And we should also look for any vars that are using the same values

**Review by SimonBrandner — APPROVED** [review]

**Review by weeman1337 — APPROVED** [review]

**Pull request merged**